zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/listeners/task/TaskListenerTest.java

```diff
@@ -259,94 +258,6 @@ public void shouldEvaluateExpressionsForTaskListeners() {
     assertThatProcessInstanceCompleted(processInstanceKey);
   }
 
-  @Test
-  public void shouldMakeVariablesFromPreviousTaskListenersAvailableToSubsequentListeners() {
```

Comment:
Thanks, Nico, for the suggestion!
I'm not a big fan of commenting out or ignoring tests that are currently irrelevant since some API might change, and even ignored tests could require additional maintenance (probably that is why I removed them a bit too easily 😅).
But the scenarios covered here should ideally still behave in the same way once we implement #23702, and the API used in these tests is stable enough.
So, I’m re-adding them with the `@Ignore` annotation as it was recommended.